summaries/2023-09-05-bitcoin-socratic-seminar-21.json

```diff
@@ -0,0 +1,100 @@
+      "summaryeli15": "Mutiny is a new wallet that lets you use Bitcoin on the internet. It's special because it runs right in your web browser without needing to download any software. It's made with a programming language called Rust and has features like sending and receiving payments. It also uses special technology to sync with the Bitcoin network quickly and keep your wallet backed up online. In the future, they might make versions of Mutiny for phones or servers too. You can try it at mutinywallet.com.",
+      "title": "Mutiny uses LDK to create the first lightning wallet for the web",
+      "link": "https://lightningdevkit.org/blog/mutiny-uses-ldk-the-first-lightning-wallet-for-the-web/"
+    },
+    {
+      "summary": "A plan to test HTLC endorsement and local reputation tracking to mitigate jamming attacks was discussed at the summit. The goals are to use real-world data for local reputation algorithms, obtain liquidity and slot utilization data, and provide a common data export format. The plan has several phases: 1) Collect anonymized forwarding data with a common format, 2) Propagate experimental 'endorsement' TLV, and 3) Implement local reputation and set 'endorsement' values. Eclair, CLN, LND, and LDK are involved in this multi-team effort. The research plan includes collecting anonymized data using a CSV format with specific fields, propagating the endorsement TLV, and implementing local reputation algorithms. The endorsement TLV is a signal to indicate endorsement and will only be used for data collection. Experimenters can use the full range of bits, but any non-zero value will be interpreted as a positive endorsement signal.",
+      "summaryeli15": "A group of people are working on a plan to test and improve a system that helps protect against attacks on a network. They want to collect data from real transactions to see how well the system works in different situations. They will use this data to set better default settings for the system. They also want to create a common format for the data so that everyone can analyze it easily. There are three steps to the plan: collect data, add a special signal to transactions, and improve the system based on the data. They are asking for help from other people who are interested in this.",
```

Comment:
```suggestion
      "summaryeli15": "A group of people are working on a plan to test and improve a system that helps protect against attacks on the lightning network. They want to collect data from real transactions to see how well the system works in different situations. They will use this data to set better default settings for the system. They also want to create a common format for the data so that everyone can analyze it easily. There are three steps to the plan: collect data, add a special signal to transactions, and improve the system based on the data. They are asking for help from other people who are interested in this.",
```